npm/private/npm_import.bzl

```diff
@@ -593,11 +593,18 @@ bin = bin_factory("node_modules")
             rctx_files[build_file].append("""exports_files(["{}"])""".format(_PACKAGE_JSON_BZL_FILENAME))
 
     rules_js_metadata = {}
+
+    custom_postinstalls = rctx.attr.custom_postinstall
+    if rctx.attr.package.startswith("@aws-sdk/client"):
```

Comment:
this was a hack, really we should check if `custom_postinstalls == ["optimize_package"]` or something